### DISCUSSION TITLE
v24.0.0-beta.2

### DISCUSSION THREAD

**Discussion by thaJeztah** [moby/moby#45340] [Announcements]
## 24.0.0-beta.2

This is a pre-release of the upcoming 24.0.0 release.

Pre-releases are intended for testing new releases: **only install in a test environment!**

```bash
curl -fsSL https://get.docker.com -o get-docker.sh
sudo CHANNEL=test sh get-docker.sh
```

### Known issues:

- There is no changelog yet; an overview of pull requests included in this release can be found on GitHub:
    - docker cli: [all pull requests for 24.0.0](https://github.com/docker/cli/pulls?q=is%3Apr+milestone%3A24.0.0+is%3Amerged) / [all "changelog" pull requests for 24.0.0](https://github.com/docker/cli/pulls?q=is%3Apr+milestone%3A24.0.0+label%3Aimpact%2Fchangelog+is%3Amerged)
    - docker engine: [all pull requests for 24.0.0](https://github.com/moby/moby/pulls?q=is%3Apr+milestone%3A24.0.0+is%3Amerged) / [all "changelog" pull requests for 24.0.0](https://github.com/moby/moby/pulls?q=is%3Apr+milestone%3A24.0.0+label%3Aimpact%2Fchangelog+is%3Amerged)
- There are no packages available yet for the s390x and ppc64le architectures
- There may be bugs! :warning:

Bugs and regressions can be reported in these issue trackers:

- Related to the CLI: https://github.com/docker/cli/issues
- Related to the Docker Engine https://github.com/moby/moby/issues

When reporting issues, include `[24.0.0-beta]` in the issue title

<hr /><em>This discussion was created from the release <a href='https://github.com/moby/moby/releases/tag/v24.0.0-beta.2'>v24.0.0-beta.2</a>.</em>

**Comment by arukiidou** [comment]
Is containerd integration the primary goal of this release? Or can we expect other major features?

**Reply by thaJeztah** [reply]
Containerd integration we made a lot of progress on (not fully complete yet), but also updated BuildKit versions, various improvements in our networking stack.

Changelog will be worked on ☺️

**Reply by arukiidou** [reply]
I got it.Thank you for your kindness!
